package.json

```diff
@@ -90,15 +90,6 @@
     "eslint": "^4.18.2",
     "eslint-plugin-react": "^7.7.0",
     "file-loader": "^1.1.6",
-    "karma": "^2.0.0",
```

Comment:
Why are these dependancies necessary?

Reply:
We were using/planning to use Karma but ended up running our tests straight from Mocha.  These aren't needed right now and were slowing down builds in the new system.  They can always be re-added in the future if we use Karma.

Reply:
Nevermind. I'm cutting back on my caffeine and I thought red was green.